load_default_pseudomonas_data.sh

```diff
@@ -24,7 +24,7 @@ python manage.py add_ml_model "Ensemble ADAGE 300" 208964 --g2g_edge_cutoff 0.4
 
 # Import activity data into the database:
 python manage.py import_activity \
-    ../data/all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt \
+    ../data/sample_signature_activity.txt \
```

Comment:
@rzelayafavila: Should this shell script also include the commands to generate the other machine learning model whose gene-gene network edge cutoff value is 0.2?

Reply:
@rzelayafavila: If this PR doesn't have to include the commands to add the machine learning model with lower gene-gene network edge cutoff value, please add an issue now. Eventually the docker deployment should be doing exactly the same thing that Fabric deployment does now.

Reply:
Ok, I will add an issue for this, so we can update this script later.